sormas-ui/src/main/java/de/symeda/sormas/ui/customexport/ExportConfigurationsGrid.java

```diff
@@ -59,6 +66,16 @@ public ExportConfigurationsGrid(
 		reload();
 	}
 
+	public ExportConfigurationsGrid(
```

Comment:
Seems like you've implemented 2 different components into one and duplicated each of the methods.

This could be solved by using some additional class fields.

Reply:
Still a lot of duplictes.